enarx-keep/src/backend/mod.rs

```diff
@@ -43,6 +44,16 @@ pub struct Datum {
     pub mesg: Option<String>,
 }
 
+#[allow(dead_code)]
+pub enum State {
+    Building(Box<dyn Builder>),
+    Complete(Arc<dyn Keep>),
```

Comment:
I think that "Complete" is likely to be a little coarse, and that we'll probably need more states after that, but this looks good for now.